1) Вы провели эксперимент c упрощением формы заказа в магазине Утконос и получили результаты по метрике конверсий в покупку. Выберите метод оценки и оцените есть ли стат.значимые различия между конверсиями в двух группах при alpha = 5%. Дайте краткие рекомендации команде. <br>Результаты: <br> - 1)Число юзеров в группах , которые заходили на сайт в период эксперимента: n1 = 15550 и n2 = 15550. <br> - 2) Число юзеров в группах , которые совершили хотя бы одну покупку за период эксперимента: n1 = 164 и n2 = 228 <br> - 3) Конверсии : conv1 = 1.05% conv2 = 1.47%. 

Для оценки статистической значимости различий между конверсиями в двух группах при α=5% можно использовать Z-критерий для разности долей.
Формула Z-критерия:

Z = (p1 - p2) / sqrt(p * (1 - p) * (1/N1 + 1/N2))

где p = (n1 * p1 + n2 * p2) / (n1 + n2)

p1 и p2 - конверсии в группах, N1 и N2 - количество юзеров в группах, которые заходили на сайт в период эксперимента, n1 и n2 - количество юзеров, которые совершили хотя бы одну покупку за период эксперимента.


In [7]:
from math import sqrt

p1 = 164 / 15550
p2 = 228 / 15550
n1 = 164
n2 = 228
N1 = 15550
N2 = N1

p =  (n1 * p1 + n2 * p2) / (n1 + n2)
Z = (p1 - p2) / sqrt(p * (1 - p) * (1/N1 + 1/N2))
Z

-3.2110930329358367

 -3.2110930329358367 < 1.96 расчетное значение Z меньше критического значения, отвергаем нулевую гипотезу о равенстве конверсий в группах. Изменения внесенные в форму дали статистически значимый эффект на уровень конверсии в покупку, в дальнейшем лучше использовать упрощенную форму.

2) Сравниваем метрику конверсия в покупку. Размер выборки - 10000 элементов в каждой группе . Какой статистический критерий тут лучше всего подойдёт и почему ?

Может быть использован z-критерий, так как элементов в каждой группе более 30